configstore/store.py

```diff
@@ -1,3 +1,15 @@
+from typing import TypeVar
+
+from .backends.env_var import EnvVarBackend
+from .backends.dotenv import DotenvBackend
+from .backends.awsssm import AwsSsmBackend
+from .backends.docker_secret import DockerSecretBackend
+
+
+Backends = TypeVar('Backends', EnvVarBackend, DotenvBackend,
+                   AwsSsmBackend, DockerSecretBackend)
```

Comment:
It’s not great to have to import all backends here.

Could we declare and interface or ABC or base type and use that?

Reply:
I woudl really prefer to declare a base class or an interface. It was my first plan, but I wasn't sure if you'd like it and I saw the TypeVar in the documentation.

My toher questions is since we'll have a backend inteface. could we move all backend in a `backends.py` file with the `Backend` class ?

Reply:
Go for an ABC or whatever is recommended by typing / mypy.

I don’t see benefits in moving the concrete backend classes though.